jobs/shield-agent/templates/bin/post-start

```diff
@@ -62,31 +75,31 @@ job() {
 EOF
 }
 <% p('schedules', {}).each do |name, schedule| -%>
-cat <<EOF | provision '<%= name %>' schedule
-{"name":"<%= name %>",
+cat <<EOF | provision '<%= format_name(name,  "schedule name") %>' schedule
+{"name":"<%= format_name(name , 'schedule name') %>",
  "when":"<%= schedule %>"}
 EOF
 <% end -%>
 
 <% p('retention-policies', {}).each do |name, retention| -%>
-cat <<EOF | provision '<%= name %>' policy
-{"name":"<%= name %>",
+cat <<EOF | provision '<%= format_name(name, "provision name") %>' policy
+{"name":"<%= format_name(name, 'provision name') %>",
```

Comment:
I believe you mean "policy" here, not "provision"

Reply:
Indeed, lets fix this.

Reply:
Done!